### DISCUSSION TITLE
InlineKeyboardButton can be clicked several times

### DISCUSSION THREAD

**Discussion by vittxr** [eternnoir/pyTelegramBotAPI#1738] [Q&A] [answered]
The user can click more than once in button and the bot process all the request he sends. Is there a easy way to avoid this? I try to "disable" (edit actually) reply_markup, but it's not helpful for me, because I need this reply markup to be enabled for other users in group. I have tried something like this: 

        `bot.edit_message_reply_markup(chat_id, call.message.message_id, reply_markup=disabledInlineKeyboard)`

 But, as I said, this code will disable the markup for all group users

**Comment by coder2020official** [comment, marked as answer]
It is Telegram Android bug, you can use class based middlewares and stop update from processing

**Comment by vittxr** [comment]
Thanks! ;). I will try to implement it
